NodeOpera/Dockerfile.txt

```diff
@@ -0,0 +1,42 @@
+USER root
+
+#=========
+# Opera
+#=========
+ENV OPERA_VERSION=60.0.3255.170
```

Comment:
Why do we need to hardcode the Opera version, can't the latest one be retrieved dynamically? Can we install any given version?

Reply:
Env variable opera version deleted. Always latest available opera will be installed.